org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/JavaSearchBugsTests.java

```diff
@@ -12081,7 +12086,7 @@ public void testBug317264b() throws CoreException {
 				IJavaSearchConstants.WAIT_UNTIL_READY_TO_SEARCH,
 				null);
 		assertSearchResults("Unexpected search results!",
-				"Enum (not open) [in Enum.class [in org.apache.commons.lang.enum [in /JavaSearchBugs/lib/b317264/org.apache.commons.lang_2.modified.jar [in P]]]]",
```

Comment:
can you explain why this is no longer found?
Otherwise this should go on the follow-up list :)

Reply:
I believe because enum was identifier at original source level and now keyword, but I will check that

Reply:
The comment on the test says `types in enum package of org.apache.commons.lang.jar should be reported for 1.4 projects`. Looking on the test `testBug317264a` above the match shouldn't be reported for 1.5 projects (which we have now). I will remove this test and some tests below that as they test now a non existing use case that can only happen on 1.4.